### PR TITLE
GRW-1755/feat/ 🚗 Add milage dropdown with prefilled values

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Added a select field to se_car with the values used in the old embark flow.

Screenshot 2022-11-09 at 11 11 48

## Justify why they are needed

Makes it easy for users to fill in their milage. 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1755]: https://hedvig.atlassian.net/browse/GRW-1755?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alebedev — APPROVED** [review]
Looks good.  Not localized, but let's focus on that at some later time

File: apps/store/src/services/PriceForm/data/SE_CAR.ts [hunk 1]
```diff
@@ -35,11 +35,17 @@ export const SE_CAR: Template = {
...
             name: 'milage',
             label: { key: 'Annual mileage' },
             required: true,
-            min: 0,
+            options: [
+              { label: { key: '1 000 mil/år' }, value: '1000' },
+              { label: { key: '1 500 mil/år' }, value: '1500' },
+              { label: { key: '2 000 mil/år' }, value: '2000' },
+              { label: { key: '2 500 mil/år' }, value: '2500' },
+              { label: { key: '2 500+ mil/år' }, value: '2501' },
```
Comment (gustaveen) [inline apps/store/src/services/PriceForm/data/SE_CAR.ts:47]: nit: seems like we use `en` as standard for untranslated values, makes it a bit nicer to demo progress if it's all in the same lang :)

**Review by gustaveen — APPROVED** [review]
Sweet! 🙌

File: apps/store/src/services/PriceForm/data/SE_CAR.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/services/PriceForm/data/SE_CAR.ts]: Realised we use kilometers for english :D 
Screenshot 2022-11-09 at 14 40 32

Comment (gustaveen) [reply apps/store/src/services/PriceForm/data/SE_CAR.ts]: value is the same though!

**Pull request merged**